### ISSUE TITLE
NoMethodError: undefined method `without' for {:expires_in=>1800, :throttle_interval=>0, :includes=>[:status]}:Hash

### ISSUE THREAD

**Issue by walter-silva** [inkstak/activejob-status#24]
Hi there! After upgrading to version 0.4.1 I'm faced with the following error:
```
NoMethodError: undefined method `without' for {:expires_in=>1800, :throttle_interval=>0, :includes=>[:status]}:Hash
Did you mean?  with_options
```
https://github.com/inkstak/activejob-status/blob/f773a9895a15886398f8ad7ea5eb9681b63a33e7/lib/activejob-status/status.rb#L12

Shouldn't the correct method be `except`? I'm not found references to this method `without` for the Hash class.

Thanks.

**Comment by inkstak** [comment]
It an ActiveSupport extension on `Enumerable` : https://api.rubyonrails.org/classes/Enumerable.html#method-i-without

What version of ActiveSupport are you using ?

**Comment by inkstak** [comment]
The `without` method has been introduced in `ActiveSupport 5.x`
The current gemspec only requires `ActiveSupport >= 4.2` dependency.
Both are in EOL.

I would be more likely to publish a new version of `activejob-status` with supported dependencies.
However, you're free to open a PR to support older versions of rails.

Last solution is to restrict your use of `activejob-status` to the `0.2.2` version

**Comment by inkstak** [comment]
I plan to deploy the following solution:

* First of all, I'm looking to add [Appraisal](https://github.com/thoughtbot/appraisal) to the test suite to test the gem against various versions of Rails gems.

* A new version `1.x` will be released with only supported dependencies (`>= 6.0`)

* I will push a `v0` branch with Appraisal set up. Any PR will be welcomed to support legacy dependencies (`>= 4.2`) and publish a last `0.5` version.

**Comment by inkstak** [comment]
New versions `v1.0`  & `v0.5` have been released.

If you need support for Rails 4.2, you should use `activejob 0.2.2`or submit a PR (see [releases](https://github.com/inkstak/activejob-status/releases))
